app/assets/stylesheets/main.css.scss

```diff
@@ -0,0 +1,8 @@
+.user-links,
+.navigation-links {
+  text-align: right;
+}
+
+#error_explanation {
+  font-weight: bold;
+}
```

Comment:
All of the other forms I have seen so far we have been using flash messages to deliver error messages see the Questions#create action for an example.  Do you want to do that here?

You will see a flash section in the application layout to hold these messages.  With that said, we have not yet implemented any styling for the flash element--you don't have to do that now, we can add a trello card for it..